src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
I would suggest rewriting it into one `filter` with a condition inside.

And use something more descriptive than just `p`. Maybe `person`?

⚠️ Fix in other functions too.

Example:
```
  const men = people.filter(person => (
    century
      ? century === Math.ceil(person.died / 100) && person.sex === 'm'
      : person.sex === 'm')
  );
```